e2e/helpers/app.js

```diff
@@ -222,6 +223,15 @@ async function prepareAndroid() {
 	await runCommand('adb shell settings put global animator_duration_scale 0.0');
 }
 
+async function expectToBeVisible(id) {
```

Comment:
Don't do this.
This is error prone, since you avoid the synchronization of Detox.
Also, why would do this anyway?
The test should assert a specific scenario.
Why would it change depending on the value?

Reply:
Deleted!